modules/ossm-installing-operator.adoc

```diff
@@ -4,7 +4,12 @@
 :_mod-docs-content-type: PROCEDURE
 [id="ossm-installing-operator_{context}"]
 = Installing the {SMProductShortName} Operator
-:context: ossm-installing-operator
+
+[WARNING]
+====
+For clusters without existing {SMProduct} instances, follow this procedure. {SMProduct} operates cluster-wide, requiring proper scope configuration to prevent conflicts between {istio} control planes. For clusters with {SMProduct} 3 or later, see xref:../install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc#ossm-deploying-multiple-service-meshes-on-single-cluster[Deploying multiple service meshes on a single cluster]
+.
+====
 
 .Prerequisites
```

Comment:
```suggestion
.Prerequisites
```

Can't comment on the text because it's not changed in this PR. But noticed in the preview that these prereqs are missing bullets `*`. You might want to consider adding them (here or in another PR)